lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -230,8 +234,15 @@ def add_3ds(post, options)
 
       def commit(action, post, authorization = nil)
         begin
-          raw_response = (action == :verify_payment ? ssl_get("#{base_url}/payments/#{post}", headers(action)) : ssl_post(url(post, action, authorization), post.to_json, headers(action)))
-          response = parse(raw_response)
+          raw_response = if action == :verify_payment
+            ssl_get("#{base_url}/payments/#{post}", headers(action))
+          elsif action == :delete_card
+            ssl_request(:delete, url(post, action, authorization), nil, headers(action))
+          else
+            ssl_post(url(post, action, authorization), post.to_json, headers(action))
+          end
```

Comment:
We had a discussion about this and it will be refactored in the next PR with partial update support

Reply:
In the partial update, we will need to use another HTTP method `patch`, and it will be better to do some refactor there.